test/libyul/yulOptimizerTests/valueConstraintBasedSimplifier/multi_assign.yul

```diff
@@ -0,0 +1,54 @@
+{
+    function f() -> x, y {}
+    let a, b := f()
```

Comment:
Are multiple declarations not being printed?

Reply:
Just assignments